assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.test.js

```diff
@@ -21,15 +21,34 @@
  */
 import KeyMetricsSetupCTAWidget from './KeyMetricsSetupCTAWidget';
 import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { MODULES_ANALYTICS_4 } from '../../modules/analytics-4/datastore/constants';
+import { MODULES_SEARCH_CONSOLE } from '../../modules/search-console/datastore/constants';
+import { getAnalytics4MockResponse } from '../../modules/analytics-4/utils/data-mock';
+import { getSearchConsoleMockResponse } from '../../modules/search-console/util/data-mock';
 import {
 	render,
 	createTestRegistry,
 	provideModules,
 	provideSiteInfo,
+	muteFetch,
 } from '../../../../tests/js/test-utils';
 
 describe( 'KeyMetricsSetupCTAWidget', () => {
```

Comment:
Should we add a couple more test cases which assert that the CTA isn't rendered when SC and/or GA4 are in a gathering data state?

Reply:
Sounds good to me! I've added a couple more tests :+1:

Reply:
Perfect, thanks!